### DISCUSSION TITLE
Anybody using Pisugar2 with this image

### DISCUSSION THREAD

**Discussion by kagehisa** [jayofelony/pwnagotchi#5] [General]
Is anybody using a Pisugar2 on a Raspberry 02W together with this image? It worked really fine on the old rRaspberry 0W and the old pwnagotchi-torch image.
It seems the required I2C Adress that is needed to read the Battery level is not present. Anybody having the same issue?

This is most liekly not an issue with the image itself, I guess the provided packages from the vendor are not working right in conlusion with Raspberry Pi OS Bookworm. I just wan't to know if anybody has a similiar experience.

**Comment by spitfire** [comment]
I am not, but I’m bumping this, as I’m [also looking ](https://github.com/jayofelony/pwnagotchi-bookworm/discussions/9)for a UPS addon.

**Reply by kagehisa** [reply]
I got it working. The [pwnagotchi plugin](https://github.com/tisboyo/pwnagotchi-pisugar2-plugin) works, but on a pi zero 2w you need to be aware that the battery level is not available when the usb ethernet bridge is active or the pi is plugged in.  Also make sure to install the power-rs manager vie the script they mention in their github and not via the old package repository. The 64 bit packages are not available via a repo.
